src/input.test.forms.ts

```diff
@@ -36,7 +36,7 @@ it('can be reset if there was no initial value', async () => {
   expect(host.value).to.be.empty.string;
 });
 
-it('has `formData` value when it has a value', async () => {
+it('has a `formData` value when it has a value', async () => {
```

Comment:
Is a `formData` a thing that a component or element has?

Reply:
Sorry for the churn. I think your edit was right. For whatever reason, I skipped over the "value" part when reading it.